posthog/warehouse/models/external_data_resource.py

```diff
@@ -0,0 +1,17 @@
+from posthog.models.utils import UUIDModel, CreatedMetaFields, sane_repr
+from django.db import models
+from posthog.models.team import Team
+
+
+class ExternalDataResource(CreatedMetaFields, UUIDModel):
```

Comment:
Sorry to be pedantic about this, though it's a nightmare to fix later: I don't think external data source is the right name. I want to make sure we're consistently using "Source", in contrast to "Destination" which we're using on the pipeline side.

A few suggestions
- `ExternalDataSource`
- `DatawarehouseSource`
- `ETLSource`

Thoughts?

Reply:
Ha, I was in the middle of changing it to option 1 (ExternalDataSource) but then did "resource" mainly because the entities that are created are (source, connection, destination). But I think it's ok to do ExternalDataSource anyway so I'll change it to that